scripts/Muon/GUI/Common/contexts/muon_group_pair_context.py

```diff
@@ -15,19 +17,37 @@
 from Muon.GUI.Common.observer_pattern import Observable
 
 
+def get_grouping_psi(workspace):
+    grouping_list = []
+
+    for ii in range(0, workspace.getNumberHistograms()):
+        sample_log_label_name = "Label Spectra " + str(ii)
+        sample_log_value = ""
+        workspace_run = workspace.getRun()
+        if workspace_run.hasProperty(sample_log_label_name):
+            sample_log_value = workspace_run.getProperty(sample_log_label_name).value
+        grouping_list.append(MuonGroup(sample_log_value, [ii+1]))
```

Comment:
should this be in the if statement, otherwise we may add a sample log of an empty string

Reply:
It probably should be.